src/main/java/uk/gov/hmcts/sptribs/ciccase/model/ContactPreferenceType.java

```diff
@@ -10,15 +10,18 @@
 public enum ContactPreferenceType implements HasLabel {
 
 
-
     @JsonProperty("Email")
     EMAIL("Email", "Email"),
     @JsonProperty("Post")
     POST("Post", "Post");
     private String type;
     private final String label;
+
     public boolean isEmail() {
         return EMAIL.name().equalsIgnoreCase(this.name());
     }
 
+    public boolean isPost() {
+        return POST.name().equalsIgnoreCase(this.name());
```

Comment:
Enums are singleton pattern - wouldn't `return POST == this;` work just as well? == is superior when checking enum equality, as it caters for nulls.